R/expect_lint.R

```diff
@@ -80,23 +80,24 @@ expect_lint <- function(content, checks, ..., file = NULL, language = "en") {
 
   local({
     itr <- 0L
```

Comment:
bah... `object_usage_linter()` false positives... let's ditch the `<<-` updates for now. Our `<<-` usage is pretty safe overall, where the initial assignment of the `<<-` target is always _just_ before the scoped function using `<<-`.